.github/workflows/markdown.links.config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "ignorePatterns": [
+		{
+			"pattern": "https://kinesis.us-east-1.amazonaws.com/"
+		},
+        {
+            "pattern": "http://localhost:4566"
+        },
+        {
+            "pattern": "http://localhost:4598"
+        },
+        {
+            "pattern": "http://my-domain.us-east-1.es.localhost.localstack.cloud:4566"
+        },
```

Comment:
according to [the docs](https://github.com/tcort/markdown-link-check#markdownlinkcheckmarkdown-opts-callback), the patterns here can be regular expressions. can we maybe generalize the localhost pattern into a regex? not sure how to express it correctly here, but something like `http://(.*).localhost.localstack.cloud(.*)`, ... or really anything with localhost in it ;-)